data/coalition/coalition missions.txt

```diff
@@ -3167,3 +3166,450 @@ mission "Heliarchs Buy Jump Drive Mission"
+			choice
+				`	(Disregard.)`
+					decline
+				`	(Look for Anna Fangvin.)`
+			`A quick look through your encyclopedia reveals Anna Fangvin to be an author from <destination>. You decide to meet her there for an interview and set a course.`
+	on complete
+		conversation
+			`As you prepare to land, you write Anna a short message, offering an interview. In less than 10 minutes you receive a reply, explaining that Anna has accepted your offer and will wait for you in the spaceport.`
+			`	Look for Anna in the spaceport, so you can take her to the Coalition for a little conversation with those Coalition citizens who showed interest in her work.`
+
+mission "Fangvin: Struggling Author 1"
+	name "Take Anna to Coalition Space"
+	description "Take Anna Fangvin to <destination>, where she might find success as an author after having struggled on <origin> for so long."
+	source "Zug"
+	destination "Ki Patek Ka"
+	to offer
+		has "Fangvin: Book Discovered: done"
+	on offer
+		conversation
+			`Upon entering the spaceport, you walk past a restaurant. A woman then raises her arms, waving towards you. She must be Anna Fangvin, that author the Coalition citizens showed interest in.`
+			choice
+				`	(Approach her.)`
+			`	You approach the woman and sit right next to her. "Hello, are you <first>?`
+			choice
+				`	"Yes, I am. So, what do you want to ask?"`
+			choice
+				`	"What do you write?"`
+					goto "what do you write"
+			label "what do you write"
+			`	"I write science fiction novels. My latest novel was 'Beyond the Hyperdrive', which I released a few weeks ago. It's about a bunch of humans who steal jump drives from the Quarg. Here, have a copy."`
+			`	Anna retrieves a hardcover book from her bag. It has an image of a Jump Drive on it. You take the book.`
+			`	"You'll like it, I promise."`
+			choice
+				`	"How long is the book?`
+					goto "how long is the book"
+				`	"Why is it a story about humanity stealing from the Quarg?"`
+					goto "anna hates quarg"
+			label "how long is the book"
+			`	It isn't very long, it should be three hundred pages.`
+			label "anna hates quarg"
+			`	"I have a lot of problems with them. I just don't trust them. I remember when my parents took me to Humanika. Naturally, I wanted to explore, so I wandered off. I was lost. This creepy-looking Quarg found me, and when it tried to approach, I cried."`
+			`	"And then when I grew up, when I was a teenager, I decided to do a project on them for college. They responded with silence to my questions, as if they had something to hide. I worked as an assistant for a journalist for a few months. He kept asking me for help with the Quarg. They just would not budge."`
+			choice
+				`	"Well that's all well and good. But I have my own business to attend to now. Good luck, and goodbye."`
```

Comment:
Probably shouldn't have a decline so early on as that makes the player seem kinda like a dick, having been the one to ask for the interview. Personally I don't think this mission specifically needs a decline, but if you do add in one should be later on.